src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,18 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    private static final int COUNT_FIGURE_IN_ARRAY = 6;
+    private static final int PIECE_OF_FIGURES_IN_ARRAY = 2;
 
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[COUNT_FIGURE_IN_ARRAY];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = i >= figures.length / PIECE_OF_FIGURES_IN_ARRAY
+                    ? figureSupplier.getDefaultFigure() : figureSupplier.getRandomFigure();
+        }
+        for (int i = 0; i < figures.length; i++) {
+            figures[i].drow();
+        }
+    }
```

Comment:
Do we actually need a separate loop? Could it affect the performance, if we had an array of billions of elements?

Let's print figures in the previous loops after their creation.

```suggestion
        for (int i = 0; i < figures.length; i++) {
            figures[i] = i >= figures.length / PIECE_OF_FIGURES_IN_ARRAY
                    ? figureSupplier.getDefaultFigure() : figureSupplier.getRandomFigure();
                    //draw
        }
      
    }
```